delete_by_query.py

```diff
@@ -5,7 +5,21 @@
 
 
 def delete_docs(es_url, index, doc_type, query):
+    """Delete documents from ES index by query.
```

Comment:
I think the function name should be `delete_by_query` or `delete_docs_by_query`.

Also, consider moving (in a separate PR) the function into separate library file, so you can both have the command-line util (current code) and a library to use programmatically:

```
from delete_by_query import delete_by_query

delete_by_query(es_connection, index, doc_type, query)
```

By the way, it is also better to have es connection as an argument rather than connecting inside the function (if we use it as a library, we probably have the connection already in our code).